tests/deploy_account.rs

```diff
@@ -0,0 +1,140 @@
+#![deny(warnings)]
+
+use felt::felt_str;
+use starknet_rs::{
+    business_logic::{
+        execution::objects::{CallInfo, CallType, TransactionExecutionInfo},
+        fact_state::in_memory_state_reader::InMemoryStateReader,
+        state::{cached_state::CachedState, state_api::State},
+        transaction::objects::internal_deploy_account::InternalDeployAccount,
+    },
+    definitions::{general_config::StarknetChainId, transaction_type::TransactionType},
+    services::api::contract_class::{ContractClass, EntryPointType},
+    utils::{felt_to_hash, Address},
+};
+use std::path::PathBuf;
+
+#[test]
+fn internal_deploy_account() {
+    let state_reader = InMemoryStateReader::new(Default::default(), Default::default());
+    let mut state = CachedState::new(state_reader, None);
+
+    state.set_contract_classes(Default::default()).unwrap();
+
+    let class_hash = felt_to_hash(&felt_str!(
+        "3146761231686369291210245479075933162526514193311043598334639064078158562617"
+    ));
+    let contract_class = ContractClass::try_from(PathBuf::from(
+        "starknet_programs/account_without_validation.json",
+    ))
+    .unwrap();
+
+    state
+        .set_contract_class(&class_hash, &contract_class)
+        .unwrap();
+
+    let internal_deploy_account = InternalDeployAccount::new(
+        class_hash,
+        0,
+        0, // TODO: Value 340282366920938463463374607431768211457 is too large.
+        0,
+        vec![
+            // TODO: Doesn't work with constructor_calldata not being empty.
+            /*felt_str!("2612640184215060329973671567201105979048071207288945836837601213263070089980")*/
+        ],
+        vec![
+            felt_str!(
+                "3233776396904427614006684968846859029149676045084089832563834729503047027074"
+            ),
+            felt_str!(
+                "707039245213420890976709143988743108543645298941971188668773816813012281203"
+            ),
+        ],
+        Address(felt_str!(
+            "2669425616857739096022668060305620640217901643963991674344872184515580705509"
+        )),
+        StarknetChainId::TestNet,
+    )
+    .unwrap();
+
+    let tx_info = internal_deploy_account
+        .execute(&mut state, &Default::default())
+        .unwrap();
+
+    assert_eq!(
+        tx_info,
+        TransactionExecutionInfo::new(
+            // TODO: Validate.
+            None,
+            // Some(CallInfo {
```

Comment:
Do you know why this values are differing?
Is there an issue or ticket created to solved it?